src/codec/encoder/HoudiniEncoder.h

```diff
@@ -42,8 +42,7 @@ struct SerializedGeometry {
 	prtx::DoubleVector coords;
 	prtx::DoubleVector normals;
 	std::vector<uint32_t> counts;
-	std::vector<uint32_t> holeCounts;
-	std::vector<uint32_t> holeIndices;
+	uint32_t holeCount;
```

Comment:
This holeCount could be a simple "hasHoles" bool

Reply:
I leave it